<launch>
	<node name="config_steering" pkg="steering" type="config_steering.py" output="screen" />
</launch>